server/base/src/main/java/org/apache/accumulo/server/manager/state/MetaDataTableScanner.java

```diff
@@ -176,8 +176,7 @@ public static TabletLocationState createTabletLocationState(Key k, Value v)
         }
         current = location;
       } else if (cf.compareTo(LogColumnFamily.NAME) == 0) {
-        String[] split = entry.getValue().toString().split("\\|")[0].split(";");
-        walogs.add(Arrays.asList(split));
+        walogs.add(Collections.singleton(LogEntry.fromMetaWalEntry(entry).getFilePath()));
```

Comment:
There's probably additional room for improvement in here, because walogs doesn't need to be a collection of collections of strings anymore (probably), but this is a sufficient fix for now. I can look at more changes later.